core/loadpoint_session.go

```diff
@@ -73,6 +73,8 @@ func (lp *Loadpoint) stopSession() {
 	s.PricePerKWh = lp.sessionEnergy.PricePerKWh()
 	s.Co2PerKWh = lp.sessionEnergy.Co2PerKWh()
 	s.ChargedEnergy = lp.sessionEnergy.TotalWh() / 1e3
+	chargedSeconds := lp.chargeDuration.Seconds()
```

Comment:
Falls `time.Duration` dann vllt. sogar `.Round(time.Minute)` runden?

Reply:
s.o. ich verstehe den Charme hier im Go Typesystem zu bleiben, aber `ns` in der DB zu halten fühlt sich wie Overkill an. Vielleicht übersehe ich hier auch was, hab aber auf die schnelle keine Konfigurationsmöglichkeiten gefunden.